app/assets/stylesheets/sass/abstracts/_variables.css.scss

```diff
@@ -0,0 +1,8 @@
+// Color variables
```

Comment:
your variable names have different formats ($primaryColor, $normal_Cyan, $font-stack)
you should use only one format